storage/tokudb/ha_tokudb_alter.cc

```diff
@@ -898,22 +1578,16 @@ bool ha_tokudb::commit_inplace_alter_table(
     bool commit) {
 
     TOKUDB_HANDLER_DBUG_ENTER("");
-    
+
     tokudb_alter_ctx* ctx =
         static_cast<tokudb_alter_ctx*>(ha_alter_info->handler_ctx);
     bool result = false; // success
     THD *thd = ha_thd();
 
     if (commit) {
-#if (50613 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50699) || \
-    (50700 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50799) || \
-    (100000 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 100099)
         if (ha_alter_info->group_commit_ctx) {
             ha_alter_info->group_commit_ctx = NULL;
         }
-#endif
-#if (50500 <= MYSQL_VERSION_ID && MYSQL_VERSION_ID <= 50599) || \
```

Comment:
Inside of this block should have been removed too?

Reply:
That is a good catch. This would have introduced a real regression.